Jenkinsfile

```diff
@@ -39,12 +40,13 @@ if ( config.branches.containsKey(env.BRANCH_NAME) ) {
                 }
             }
             utils.ircNotification(config, [stage: 'Test & Deploy', status: 'starting'])
-            try {
-                utils.buildDockerImage(dockerfile: 'bedrock_base', update: true)
-                utils.buildDockerImage(dockerfile: 'bedrock_code', fromDockerfile: 'bedrock_base')
-            } catch(err) {
-                utils.ircNotification(config, [stage: 'Docker Build', status: 'failure'])
-                throw err
+            lock ("bedrock-docker-${env.GIT_COMMIT}") {
```

Comment:
This is to make sure that the same commit pushed to 2 (or more) branches is only built once. This shouldn't happen often of course, but it does allow us to push `master`, then tag and push `prod` without much fear.